gui/default/assets/lang/lang-en.json

```diff
@@ -228,6 +228,7 @@
    "Version": "Version",
    "Versions Path": "Versions Path",
    "Versions are automatically deleted if they are older than the maximum age or exceed the number of files allowed in an interval.": "Versions are automatically deleted if they are older than the maximum age or exceed the number of files allowed in an interval.",
+   "Warning, this path is already a subdirectory of {%otherPath%}.": "Warning, this path is already a subdirectory of {{otherPath}}.",
```

Comment:
It would probably be more useful to display the label of the other folder or have this rephrased, as the way it stands now it's just telling me that 2+2=4.

Yes, I know that C:\A\B is a subdirectory of C:\A, though I might have forgotten that C:\A is an actual syncthing folder I added a week ago.